### ISSUE TITLE
3. Rename tutorial

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/BoostGraphTutorial#67]
From [a discussion](https://github.com/mywtfmp3/BoostGraphTutorial/issues/7#issuecomment-388018510).

**Comment by richelbilderbeek** [comment]
Renamed the tutorial.

**Comment by richelbilderbeek** [comment]
@mywtfmp3: I suggest to rename this GitHub to `boost_graph_cookbook_1` (snake case is more in fashion these days), also suggest to rename your GitHub to `boost_graph_cookbook_2`.

Agree? What name do you suggest otherwise?

**Comment by mywtfmp3** [comment]
I created a repo  named `boost_graph_cookbook_2`. You are added as a collaborator.

**Comment by richelbilderbeek** [comment]
Great! Thanks!

**Comment by richelbilderbeek** [comment]
@mywtfmp3 : Due to your forks: do I have a go to rename `BoostGraphTutorial` to `boost_graph_cookbook_1`?

**Comment by mywtfmp3** [comment]
I think it's better to not rename `BoostGraphTutorial` at the moment. The old name is already known to people. Maybe just add a comment in README.

**Comment by richelbilderbeek** [comment]
Agreed. That's why I suggested to rename the current one to `boost_graph_cookbook_1` (to keep the full history) *and* create a new repo called `BoostGraphTutorial`. The fresh `BoostGraphTutorial` contains only a README forwarding the users to `boost_graph_cookbook_1`.

Do you think that would be a good idea?

**Comment by mywtfmp3** [comment]
Yeah. I like that.
